mandatory/2-function-creation.js

```diff
@@ -15,26 +23,63 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  let fullDiscount = (totalPrice * 10)/100;
+  let smallDiscount = (totalPrice * 5)/100;
+  if (totalPrice >= 200){
+  return newPrice = totalPrice - fullDiscount;
+  } else {
+  return newPrice = totalPrice - smallDiscount;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
-  */
-function printOddNumbers(limit) {}
+*/
+
+
+
+function printOddNumbers(limit) {
+  let lowNumber = 1;
+  while (lowNumber < limit) {
+    console.log(lowNumber);
+  lowNumber = lowNumber + 2;
```

Comment:
looks good! 
a shorter way of writing (but no different do what you did) is
`lowNumber += 2`